browser/ui/webui/brave_vpn/vpn_panel_handler.cc

```diff
@@ -11,11 +11,9 @@
 
```

Comment:
nit: can be deleted above desktop header.

Reply:
ah, missed this one!